WcaOnRails/app/models/registration.rb

```diff
@@ -54,13 +54,7 @@ def gender
   end
 
   def countryId
-    if user
-      country = Country.find_by_iso2(user.country_iso2)
-      if country
-        return country.id
-      end
-    end
-    read_attribute(:countryId)
+    user&.country&.id || read_attribute(:countryId)
```

Comment:
Couldn't we rewrite the few idioms [below](https://github.com/cubing/worldcubeassociation.org/pull/908/files#diff-26f54392be8933bbfcbe63d0011d19c8R74) from:
```ruby
model ? model.method : something
```
to:
```ruby
model&.method || something
```
?
(or just `model&.method` when `something` is `nil`)